Sources/LiveViewNative/Utils/KeyEquivalent.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  KeyEquivalent.swift
+//  
+//
+//  Created by murtza on 24/05/2023.
+//
+
+import SwiftUI
+
+/// The key equivalent that the user presses in conjunction with any specified modifier keys to activate the shortcut.
+/// One of the `KeyEquivalent` enumerations.
+///
+/// Possible values:
+/// * `up_arrow`
+/// * `down_arrow`
+/// * `left_arrow`
+/// * `right_arrow`
+/// * `clear`
+/// * `delete`
+/// * `end`
+/// * `escape`
+/// * `home`
+/// * `page_up`
+/// * `page_down`
+/// * `return`
+/// * `space`
+/// * `tab`
+/// * `a-z`
+/// * `A-Z`
+/// * `0-9`
+/// * `!@#$%^*()-_=+[]{}|:"'<>,./?`
+#if os(iOS) || os(macOS)
```

Comment:
The doc comment needs to go inside the `#if os(...)`
```suggestion
#if os(iOS) || os(macOS)
/// The key equivalent that the user presses in conjunction with any specified modifier keys to activate the shortcut.
/// One of the `KeyEquivalent` enumerations.
///
/// Possible values:
/// * `up_arrow`
/// * `down_arrow`
/// * `left_arrow`
/// * `right_arrow`
/// * `clear`
/// * `delete`
/// * `end`
/// * `escape`
/// * `home`
/// * `page_up`
/// * `page_down`
/// * `return`
/// * `space`
/// * `tab`
/// * `a-z`
/// * `A-Z`
/// * `0-9`
/// * `!@#$%^*()-_=+[]{}|:"'<>,./?`
```